addons/project/views/project_views.xml

```diff
@@ -1240,11 +1309,11 @@
             <field eval="2" name="priority"/>
             <field name="arch" type="xml">
                 <calendar date_start="date_deadline" string="Tasks" mode="month" 
-                          color="user_ids" event_limit="5" hide_time="true" 
+                          color="color" event_limit="5" hide_time="true" 
                           event_open_popup="true" quick_add="false" show_unusual_days="True"
                           js_class="project_calendar"
                           scales="month,year">
-                    <field name="project_id" widget="project_private_task" filters="1"/>
+                    <field name="project_id" widget="project_private_task" color="color"/>
```

Comment:
Is this `color` or rather `project_color`?

Reply:
It is `color`, but I will move it to the `view_task_all_calendar` override, as it is confusing to set the color without adding the filter first